src/components/nav/MusicBar2.vue

```diff
@@ -439,6 +479,26 @@ export default {
             });
         },
         songBuffering() {},
+        mathFloorTime() {
+            return Math.floor(this.songProgressTimestamp);
+        },
+        timeFormat() {
+            const Stringtime = formatDuration(Math.floor(this.songProgressTimestamp * 1000));
+            return Stringtime;
+        },
+        mouseOver() {
+            // dont turn to !this.showNavbar and !this.MouseEnter cause it trigger multiple times in their child element too
+            this.coolDown = 0;
+            this.showNavbar = true;
+            this.MouseEnter = true;
+            console.log("Over");
```

Comment:
this is triggering multiple times for different components inside the MusicBar when i'm moving my mouse over it... kinda weird.

Reply:
It trigger all/in parrent component that cover 
I will try to find another way to make it trigger only one